src/components/PlantForm.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import axios from 'axios';
+
+import ImageResults from './ImageResults';
+
+class PlantForm extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            plantName: '',
+            description: '',
+            location: '',
+            apiUrl: 'https://pixabay.com/api',
+            apiKey: '11850512-34a79ee2d04c5d7e18f774944',
+            amount: 5,
+            image: '',
+            images: []
+        }
+    }
+
+    changeHandler = e => {
+        const val = e.target.value;
+        this.setState({[e.target.name]: val }, () => {
+            if(val === '') {
+                this.setState({images: []})
+            } else {
+            axios
+                .get(`${this.state.apiUrl}/?key=${this.state.apiKey}
+                    &q=${this.state.plantName}&image_type=photo&per_page=${this.state.amount}
```

Comment:
`&q=plant,${this.state.plantName}

will add the keyword "plant" to every search query ... i found that helps return more relevant photos when using unsplash api. Try to get a POST /api/plants fired to the backend before standup tomorrow.